src/main/java/core/basesyntax/FigureDrawing.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface FigureDrawing {
```

Comment:
```suggestion
public interface Drawer {
```

as we can use this interface not only for Figures